tests/atomizer.js

```diff
@@ -307,6 +307,53 @@ describe('getConfig()', function () {
         config = atomizer.getConfig(classNames, defaultConfig);
         expect(config).to.deep.equal(atomizer.mergeConfigs([defaultConfig, expectedConfig]));
     });
+
+    it("should append the default unit", function () {
+        var classNames = ['P-1', 'M-0.3', 'P-2%', 'C-333'];
+
+        var defaultConfig = {
+            defaultUnit: 'px',
+            margin: {
+                defaultUnit: 'em'
```

Comment:
I think that for react components the best practice is not to use this feature and keep units explicit. But for applications this feature is very useful to enforce consistency. and remove a bit of duplication.